std/array.d

```diff
@@ -163,7 +163,7 @@ if (isIterable!Range && !isNarrowString!Range && !isInfinite!Range)
     struct Foo
     {
         int a;
-        void opAssign(Foo foo)
+        void opAssign(Foo)
```

Comment:
If you do not use the parameter, yes. It is especially useful when using design patterns (for example the visitor pattern).